.github/dependency-review-config.yml

```diff
@@ -1,7 +1,9 @@
 fail-on-severity: 'high'
 allow-ghsas:
   # dependency review does not allow specific file exclusions
-  # we use an older version of NextJS in our tests and thus need to 
+  # we use an older version of NextJS in our tests and thus need to
   # exclude this
   # once our minimum supported version is over 14.1.1 this can be removed
   - GHSA-fr5h-rqp8-mj6g
+  # we need this for an E2E test for the minimum required version of Nuxt 3.7.0
+  - GHSA-v784-fjjh-f8r4
```

Comment:
Thanks for adding this 🙌 

There are two vulnerabilities so we would need to add another one: `GHSA-vf6r-87q4-2vjf`

See here: https://github.com/getsentry/sentry-javascript/actions/runs/12019453468/job/33506044190?pr=14473#step:4:37

Reply:
I think this test only fails for vulnerabilities that are marked high or critical so we can ignore the other one 👍